pdns/dnsparser.cc

```diff
@@ -295,20 +296,29 @@ void MOADNSParser::init(bool query, const char *packet, unsigned int len)
 
       d_answers.push_back(make_pair(dr, pr.d_pos));
 
+      /* XXX: XPF records should be allowed after TSIG as soon as the actual XPF option code has been assigned:
```

Comment:
Why would assignment change this rule?

Reply:
Because this function is used in the auth, rec and dnsdist so using a variable is painful, while of course not impossible. This becomes very easy once we have a fixed value to allow.